### PR TITLE
new(ci): updated workflow configs to generate docker images.

### PR DESCRIPTION
**Pull request by cappellinsamuele**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area ci

**What this PR does / why we need it**:

This PR updates the `goreleaser.yml` and the `release` gha to generate Docker images for the artifacts.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by FedeDP** [comment]
This is huge! Thank you @cappellinsamuele!

**Comment by FedeDP** [comment]
Can you also update README stating that latest release is available under `falcosecurity/syscalls-bumper:latest` dockerhub tag?

**Comment by cappellinsamuele** [comment]
> Can you also update README stating that latest release is available under `falcosecurity/syscalls-bumper:latest` dockerhub tag?

Sure!

File: README.md [hunk 1]
```diff
@@ -2,10 +2,14 @@
 
 [![CI Build](https://github.com/FedeDP/syscalls-bumper/actions/workflows/ci.yml/badge.svg)](https://github.com/FedeDP/syscalls-bumper/actions/workflows/ci.yml)
 [![Latest](https://img.shields.io/github/v/release/FedeDP/syscalls-bumper?style=flat)](https://github.com/FedeDP/syscalls-bumper/releases/latest)
+[![Docker]()]
```
Comment (FedeDP) [inline README.md:5]: Leftover? :)

Comment (cappellinsamuele) [reply README.md:5]: Yep :)

File: .goreleaser.yml [hunk 2]
```diff
@@ -14,6 +14,65 @@ builds:
...
+    image_templates:
+      - "falcosecurity/syscalls-bumper:latest-amd64"
+      - "falcosecurity/syscalls-bumper:{{ .Version }}-amd64"
+    build_flag_templates:
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+    extra_files:
```
Comment (FedeDP) [inline .goreleaser.yml:31]: I think that `extra_files` is wrong; in syscalls-bumper, it should be just:
```
.git
go.mod
go.sum
main.go
Makefile
```
i think.

File: .goreleaser.yml [hunk 3]
```diff
@@ -14,6 +14,65 @@ builds:
...
+      - "falcosecurity/syscalls-bumper:latest-arm64v8"
+      - "falcosecurity/syscalls-bumper:{{ .Version }}-arm64v8"
+    build_flag_templates:
+      - "--platform=linux/arm64/v8"
+      - "--pull"
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.name={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+    extra_files:
```
Comment (FedeDP) [inline .goreleaser.yml:55]: Same as above!

**Review by FedeDP — CHANGES_REQUESTED** [review]

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**